sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -3469,6 +3491,40 @@ public partial class EncryptionSettingsGroup
         public static bool operator !=(Azure.ResourceManager.Compute.Models.ExecutionState left, Azure.ResourceManager.Compute.Models.ExecutionState right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ExpandTypeForListVm : System.IEquatable<Azure.ResourceManager.Compute.Models.ExpandTypeForListVm>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ExpandTypeForListVm(string value) { throw null; }
+        public static Azure.ResourceManager.Compute.Models.ExpandTypeForListVm InstanceView { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Compute.Models.ExpandTypeForListVm other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Compute.Models.ExpandTypeForListVm left, Azure.ResourceManager.Compute.Models.ExpandTypeForListVm right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Compute.Models.ExpandTypeForListVm (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Compute.Models.ExpandTypeForListVm left, Azure.ResourceManager.Compute.Models.ExpandTypeForListVm right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ExpandTypesForListVm : System.IEquatable<Azure.ResourceManager.Compute.Models.ExpandTypesForListVm>
```

Comment:
We have two of these similar enums. I assume one of them is for the list by RG and the other is for list by Subs.
Is there a possibility that these two could diverge in the future? I suggest we combine them together.
You could just rename this to `GetVirtualMachinesExpandType` as well and the generator will automatically combine them

Reply:
sounds good